openbts/tests/sipauthserve_component_tests.py

```diff
@@ -359,6 +359,18 @@ def test_update_nonexistent_subscriber_by_imsi(self):
     self.assertTrue(self.sipauthserve_connection.socket.recv.called)
     self.assertEqual(response.code, SuccessCode.OK)
 
+  def test_delete_subscriber_when_sqlite_unavailable(self):
+    """Testing the subscriber deletion case when OpenBTS reports sqlite is unavailble"""
+    self.sipauthserve_connection.socket.recv.return_value = json.dumps({
+      'code': 503,
+      'data': { 'sip_buddies': 'something bad',
```

Comment:
This goes back to [https://github.com/RangeNetworks/NodeManager/blob/master/JSONDB/JSONDB.cpp#L285}(how NodeManager handles unsubscription). Since it removes the entries from both sip_buddies and  dialdata_table, it gives you the aggregate operation results in the response. Normally if both succeed it just says "both ok". In the even that one fails, it gives you the individual results for each of the queries. Point is, it allows you to know which of the individual delete operations failed.